code/package.json

```diff
@@ -46,8 +46,8 @@
     "lint:prettier": "prettier --write",
     "local-registry": "yarn --cwd ../scripts local-registry",
     "publish-sandboxes": "yarn --cwd ../scripts publish",
-    "storybook:ui": "NODE_OPTIONS=\"--preserve-symlinks --preserve-symlinks-main\" ./core/bin/index.cjs dev --port 6006 --config-dir ./.storybook",
-    "storybook:ui:build": "NODE_OPTIONS=\"--preserve-symlinks --preserve-symlinks-main\" ./core/bin/index.cjs build --config-dir ./.storybook --webpack-stats-json",
+    "storybook:ui": "./core/bin/index.cjs dev --port 6006 --config-dir ./.storybook",
```

Comment:
I thought preserving symlinks was important because most of the stories are symlinked 🤔

Reply:
They aren't symlinked in our internal storybook, only on the sandboxes.
And the sandboxes work fine as far as I can tell.